ui/webui/src/components/installation/InstallationProgress.jsx

```diff
@@ -59,6 +58,25 @@ export class InstallationProgress extends React.Component {
                                             this.props.onAddErrorNotification
                                         );
                             }
+                            console.log("MESSAGE");
+                            console.log(step);
+                            console.log(message);
+                            // storage
+                            if (step <= 3) {
+                                this.setState({ currentProgressStep: 0 });
+                            }
+                            // payload
+                            if (step === 4) {
+                                this.setState({ currentProgressStep: 1 });
+                            }
+                            // configuration
+                            if (step >= 5 && step <= 11) {
+                                this.setState({ currentProgressStep: 2 });
+                            }
+                            // bootloader
```

Comment:
Is the bootloader step really the last step? What about copying logs, for example?
I'm just wondering if bootloader configuration really deserves it's own step, IMO it's just one of the "configuration" substeps, like creating the users or generating initramfs.

Reply:
Agreed, this is just implemented based on the mockup, which has bootloader as the last step. AFAIK we have not yet really discussed the step groupings in detail and the current progress steps should be thus considered preliminary. :)

Reply:
Bootloader currently generates something and takes forever, so it makes some sense to have it as a major thing.